pyfiglet/__init__.py

```diff
@@ -57,11 +57,23 @@
 
 RESET_COLORS = b'\033[0m'
 
-if sys.platform == 'win32':
-    SHARED_DIRECTORY = os.path.join(os.environ["APPDATA"], "pyfiglet")
-else:
-    SHARED_DIRECTORY = '/usr/local/share/pyfiglet/'
-
+SHARED_DIRECTORIES = [
+    '/usr/local/share/figlet',
+    '/usr/local/share/pyfiglet',
+    '/usr/share/figlet',
+    '/usr/share/pyfiglet',
+    # add figlet/pyfiglet shared directory path
+]
+
+def find_shared_dir():
+    path = ''
```

Comment:
This default  could be problematic as anyone using this function expects a full path.    I think it should default to the original value before these changes to prevent bugs where the fonts are lost when users change directory.